### ISSUE TITLE
"IP allow list" checkbox options are affecting the JIRA and GitHub App Configuration

### ISSUE THREAD

**Issue by sherylxiyueliu** [atlassian/github-for-jira#604]
Hi, we recently found out that the GitHub App configuration in JIRA has error. Our IT team tried to debug and un-checked "IP allow list" options, which helped to make the configuration successfully. But if we enable the "IP allow list" options, the error occurred again. (Please see the attached screenshot). 

<img width="1186" alt="Screen Shot 2021-08-12 at 10 03 18" src="https://user-images.githubusercontent.com/32909809/129238536-518dc9db-b15d-46f1-a283-6a90b139b2bd.png">

Info provided by Atlassian support: 

"I just found out that there was a recent change in our IP ranges that need to allow Atlassian products to be able to communicate with third-party integrated apps. So, if you want to continue to use the IP allowlist on the Github site, you will need to ensure our IP ranges are included in that list.
Please have them modify the IP allowlist to included all of our updated IP ranges provided here: https://ip-ranges.atlassian.com/"

Could you please help to modify the IP allowlist?

Thanks!

**Comment by thombergs** [comment]
Hey @sherylxiyueliu,

This issue is already tracked here: https://github.com/atlassian/github-for-jira/issues/411. Please follow that issue, I'm going to close this as a duplicate. But here's an explanation:

The app currently doesn't support IP allowlists, because it's running in an environment that doesn't guarantee a fixed range of IP addresses. The good news is that this is going to change, hopefully during the next weekend (August 22nd), on which we're moving the app into a new environment.

Once that move is done, it should work with IP allowlist enabled. You won't even have to add the IPs manually as we'll configure them into the GitHub app and when you click the "Enable IP allowlist configuration for installed GitHub apps" checkbox, it should automatically configure those IP addresses for you:

![image](https://user-images.githubusercontent.com/1363578/129263405-55783b56-5e5d-4771-9f55-132e2fbfbc70.png)

**Comment by sherylxiyueliu** [comment]
Thanks @thombergs
